components/Network/NetworkList.tsx

```diff
@@ -1,17 +1,17 @@
 import React, { FunctionComponent } from 'react'
-import { Network } from '@prisma/client'
+import { NetworkWithConnectionInfo } from 'services/networkService'
 import style from './network.module.css'
 
-interface IProps { networks: Network[] }
+interface IProps { networks: NetworkWithConnectionInfo[] }
 export default ({ networks }: IProps): FunctionComponent<IProps> => {
   return (
     <div className={style.network_ctn}>
       {networks.map(network => (
         <div key={network.id} className={style.card_wrapper}>
             <div className={style.network_card_text}>
               <div className={style.cardTitle}>{network.title}</div>
-              <div>Last block: X time ago</div>
-                {network.title === 'eCash' ? 
+              <div>Last block: {network.minutesSinceLastBlock ?? '-'} time ago</div>
```

Comment:
Change to `minutes ago`.
In the future, we could pass the time of the last block, and let `moment()` do the rest.

Reply:
Addressed. I don't understand the second part of the comment tho

Reply:
`moment.js` is one of our projects' dependencies. It has a cool feature that accepts a date, and returns a string like `X minutes ago`
https://momentjs.com/docs/#/displaying/fromnow/

I'll make a separate task for it